docker-imagebuilder.sh

```diff
@@ -6,40 +6,43 @@ set -x
 BRANCHES="${BRANCHES:-master}"
 DOCKER_IMAGE="${DOCKER_IMAGE:-openwrt-imagebuilder}"
+DISTROS="alpine debian"
 
 for TARGET in $TARGETS ; do
-    export IMAGEBUILDER_FILE="openwrt-imagebuilder*x86_64.tar.xz"
-    for BRANCH in $BRANCHES; do
-        if [ "$BRANCH" == "master" ]; then
-            export IMAGEBUILDER_PATH="snapshots/targets/$(echo $TARGET | tr '-' '/')"
-        else
-            export IMAGEBUILDER_PATH="releases/$BRANCH/targets/$(echo $TARGET | tr '-' '/')"
-        fi
-
-        curl "https://downloads.openwrt.org/$IMAGEBUILDER_PATH/sha256sums" -sS -o sha256sums
-        curl "https://downloads.openwrt.org/$IMAGEBUILDER_PATH/sha256sums.asc" -sS -o sha256sums.asc
-        gpg --with-fingerprint --verify sha256sums.asc sha256sums
-        rsync -av "downloads.openwrt.org::downloads/$IMAGEBUILDER_PATH/$IMAGEBUILDER_FILE" . || contine # skip uploading if no IB is available
-        grep openwrt-imagebuilder sha256sums > sha256sums_imagebuilder
-        sha256sum -c sha256sums_imagebuilder
-
-        mkdir -p ./imagebuilder
-        tar Jxf $IMAGEBUILDER_FILE --strip=1 -C ./imagebuilder
-        rm -rf $IMAGEBUILDER_FILE
-
-        docker build -t "$DOCKER_IMAGE:$TARGET-$BRANCH" -f Dockerfile.imagebuilder .
-
-        rm -rf ./imagebuilder
-
-        if [ "$BRANCH" == "master" ]; then
-            docker tag "$DOCKER_IMAGE:$TARGET-$BRANCH" "$DOCKER_IMAGE:$TARGET"
-            docker push "$DOCKER_IMAGE:$TARGET"
-            if [ "$TARGET" == "x86-64" ]; then
-                docker tag "$DOCKER_IMAGE:$TARGET-$BRANCH" "$DOCKER_IMAGE:latest"
-                docker push "$DOCKER_IMAGE:latest"
+    for DISTRIBUTION in "${DISTROS}" ; do
+        export IMAGEBUILDER_FILE="openwrt-imagebuilder*x86_64.tar.xz"
+        for BRANCH in $BRANCHES; do
+            if [ "$BRANCH" == "master" ]; then
+                export IMAGEBUILDER_PATH="snapshots/targets/$(echo $TARGET | tr '-' '/')"
+            else
+                export IMAGEBUILDER_PATH="releases/$BRANCH/targets/$(echo $TARGET | tr '-' '/')"
             fi
-        else
-            docker push "$DOCKER_IMAGE:$TARGET-$BRANCH"
-        fi
+
+            curl "https://downloads.openwrt.org/$IMAGEBUILDER_PATH/sha256sums" -sS -o sha256sums
+            curl "https://downloads.openwrt.org/$IMAGEBUILDER_PATH/sha256sums.asc" -sS -o sha256sums.asc
+            gpg --with-fingerprint --verify sha256sums.asc sha256sums
+            rsync -av "downloads.openwrt.org::downloads/$IMAGEBUILDER_PATH/$IMAGEBUILDER_FILE" . || contine # skip uploading if no IB is available
+            grep openwrt-imagebuilder sha256sums > sha256sums_imagebuilder
+            sha256sum -c sha256sums_imagebuilder
+
+            mkdir -p ./imagebuilder
+            tar Jxf $IMAGEBUILDER_FILE --strip=1 -C ./imagebuilder
+            rm -rf $IMAGEBUILDER_FILE
+
+            docker build -t "$DOCKER_IMAGE:$TARGET-$BRANCH" -f imagebuilder/"Dockerfile.${DISTRIBUTION}" .
+
+            rm -rf ./imagebuilder
```

Comment:
This removes the folder containing the ImageBuilder Dockerfiles, as the IB archive is also extracted to this folder. I guess we have to choose different folder names. Same happens with the SDK